extensions/interactions/DragAndDropSortInput/DragAndDropSortInput.py

```diff
@@ -79,3 +81,203 @@ class DragAndDropSortInput(base.BaseInteraction):
     }]
 
     _answer_visualization_specs = []
+
+    @classmethod
+    def to_proto(cls, interaction):
+        """Creates a DragAndDropSortInputInstance proto object.
+
+        Args:
+            interaction: InteractionInstance. The interaction instance
+                associated with this state.
+
+        Returns:
+            DragAndDropSortInputInstance. The
+            DragAndDropSortInputInstance proto object.
+        """
+        customization_args_proto = (
+            cls._to_customization_args_proto(
+                interaction.customization_args)
+        )
+
+        outcome_proto = interaction.default_outcome.to_proto()
```

Comment:
If outcome, hints, solution, etc. will be the same for every interaction, don't do that part here. Only do the customization_args part within the interaction and do the "common stuff" in core.

Reply:
The `to_proto()` for Outcome is inside the `class Outcome` in the `state_domain` file, so it's a common thing.